### PR TITLE
Add glance tests for Cinder Storage backend

### PR DESCRIPTION
**Pull request by hemanthnakkina**
Add zaza tests for charm glance when cinder is configured
as storage backend.

Related-Bug: LP#1905042

### CODE REVIEW

File: zaza/openstack/charm_tests/glance/tests.py [hunk 1]
```diff
@@ -222,3 +223,51 @@ def test_100_create_delete_image(self):
         )
         self.assertEqual(image["size"], response["ContentLength"])
         openstack_utils.delete_image(self.glance_client, image["id"])
+
+
+class GlanceCinderBackendTest(test_utils.OpenStackBaseTest):
+    """Encapsulate glance tests using an external S3 backend."""
```
Comment (dosaboy) [inline zaza/openstack/charm_tests/glance/tests.py:229]: "external S3 backend" should be "cinder backend"?

**Review by dosaboy — COMMENTED** [review]
small nit otherwise lgtm

File: zaza/openstack/charm_tests/glance/tests.py (hunk 1 above)
Comment (hemanthnakkina) [reply zaza/openstack/charm_tests/glance/tests.py]: Fixed in the 3rd commit

**Review by nobuto-m — CHANGES_REQUESTED** [review]
I suppose this commit can be omitted by rebasing to master to include:
https://github.com/openstack-charmers/zaza-openstack-tests/pull/670

**Pull request closed without merging**